frontend/src/v5/ui/components/viewer/cards/tickets/pinDetails/pinDetails.styles.ts

```diff
@@ -18,17 +18,25 @@
 import { Typography } from '@controls/typography';
 import styled, { css } from 'styled-components';
 
-export const PinContainer = styled.div<{ selected: boolean; }>`
+export const PinContainer = styled.div<{ selected: boolean, error: boolean }>`
 	width: auto;
 	color: ${({ theme }) => theme.palette.secondary.main};
 	border: 1px solid ${({ theme }) => theme.palette.secondary.lightest};
 	border-radius: 5px;
 	padding: 10px 15px;
 	margin: 4px 0;
+
 	${({ selected, theme: { palette } }) => selected && css`
 		box-shadow: 0 0 4px ${palette.primary.main};
 		border-color: ${palette.primary.main};
 	`}
+
+	${({ error, theme: { palette } }) => error && css`
+		color: ${palette.error.main};
```

Comment:
sorry, this error stuff should have been done in my pin issue. You might also want to set the colour in SettingLocationText to stop it turning red on error. Also the error text might need some repositioning

Reply:
tbh not a big deal, wont fix.

Reply:
The whole feature will be review by george, if there's something missing he will point it out.